bindata/oauth-openshift/deployment.yaml

```diff
@@ -63,15 +63,21 @@ spec:
               fi
               exec oauth-server osinserver \
               --config=/var/config/system/configmaps/v4-0-config-system-cliconfig/v4-0-config-system-cliconfig \
-              --v=${LOG_LEVEL}
+              --v=${LOG_LEVEL} \
+              ${SERVER_ARGUMENTS}
           ports:
             - name: https
               containerPort: 6443
               protocol: TCP
           securityContext:
+            privileged: true
             readOnlyRootFilesystem: false # because of the `cp` in args
             runAsUser: 0 # because /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem is only writable by root
```

Comment:
@stlaz, @s-urbaniak, `runAsUser: 0` is already set. As I tested the `privileged: false` value before, I wouldn't do it again.

Reply:
sounds good then 👍